knn-perf-tool.py

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# The OpenSearch Contributors require contributions made to
```

Comment:
I am not sure whether we should use OpenSearch Contributors here? My opinion is, just go with Apache 2.0

Reply:
Yes. I think that makes sense.

Reply:
I've updated the license headers to be Apache 2.0